static/frontary/theme.css

```diff
@@ -2848,3 +2870,7 @@ td.tab-menu-right {
     filter: brightness(0.5);
     pointer-events: none;
 }
+
+/* .single-select-selected {
+    filter: brightness(0.25);
+} */
```

Comment:
If this is not needed, remove it please.

Reply:
Done